controller/backing_image_controller.go

```
@@ -310,22 +345,604 @@ func (bic *BackingImageController) syncBackingImage(key string) (err error) {
+		}
+	}
+
+	return nil
+}
+
+func (bic *BackingImageController) handleV2BackingImage(bi *longhorn.BackingImage) (err error) {
+	if err := bic.prepareFirstV2Copy(bi); err != nil {
+		return errors.Wrapf(err, "failed to prepare the first v2 backing image")
+	}
+
+	if bic.syncV2StatusWithInstanceManager(bi); err != nil {
```

golangci-lint (1.62.2)

Error return value of `bic.syncV2StatusWithInstanceManager` is not checked

(errcheck)